student/views.py

```diff
@@ -1,9 +1,26 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
 from home.decorators import group_required
-from home.models.models import Announcement
+from home.models.models import Announcement, Form
+import os
+from django.conf import settings
+from django.http import HttpResponse, Http404
 
 
 @group_required("student")
 def student(request):
     announcements = Announcement.objects.filter(recipient_groups=1)
```

Comment:
We probably ought to have a way to say `recipient_groups=STUDENT` or something like that so we don't directly have database ids in the code. Do we have a migration that guarantees these groups have the correct group ids?

Reply:
Fixed in 990f6f85c4e11c5e79a698e3a9999164c288cd91, look Ok?